### PR TITLE
[Docs] vis doc

### PR DESCRIPTION
**Pull request by Harold-lkk**

### CODE REVIEW

File: docs/en/user_guides/visualization.md [hunk 1]
```diff
@@ -1 +1,108 @@
 # Visualization
+
+Before reading this tutorial, it is recommended to read MMEngine's [Visualization](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/tutorials/visualization.md) documentation to get a first glimpse of the `Visualizer` definition and usage.
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:3]: This links to the Chinese doc of MMEngine.

File: docs/en/user_guides/visualization.md [hunk 2]
```diff
@@ -1 +1,108 @@
...
+- The `VisualizerHook` is implemented in MMOCR, which uses the `Visualizer` to visualize or store the prediction results of the validation or prediction phase into the backend set by the `Visualizer`, so by modifying the `Visualizer` backend in the configuration file, for example, to ` TensorBoardVISBackend` or `WandbVISBackend`, you can implement storing the predicted images to `TensorBoard` or `Wandb`.
+
+## Configuration
+
+Thanks to the use of the registration mechanism, in MMOCR we can set the behavior of the visualizer `Visualizer` by modifying the configuration file. Usually, we define the default configuration for the visualizer in `task/_base_/default_runtime.py`, see [configuration tutorial](config.md) for details.
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:19]:
```suggestion
Thanks to the use of the registration mechanism, in MMOCR we can set the behavior of the `Visualizer` by modifying the configuration file. Usually, we define the default configuration for the visualizer in `task/_base_/default_runtime.py`, see [configuration tutorial](config.md) for details.
```

File: docs/en/user_guides/visualization.md [hunk 3]
```diff
@@ -1 +1,108 @@
...
+visualization = _base_.default_hooks.visualization
+visualization.update(
+    dict(enable=True, show=True, draw_gt=False, draw_pred=True))
+```
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/24622904/187428385-e6a23120-6445-4c55-a265-c550da692087.png" height="300"/>
+</div>
+
+The `` test.py` procedure is further simplified by providing the  ``-show``` ' and ``-show-dir ``` parameters to visualize the annotation and prediction results during the test without modifying the configuration.
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:96]:
```suggestion
The `test.py` procedure is further simplified by providing the  `-show` and `-show-dir` parameters to visualize the annotation and prediction results during the test without modifying the configuration.
```

File: docs/en/user_guides/visualization.md [hunk 4]
```diff
@@ -1 +1,108 @@
...
+| data-root       | str   | The root directory where images and annotations are stored                                                                                                                           |
+| ann-file        | str   | The directory where the annotations are stored. This can be either a relative path or an absolute path, if relative then the final ann_file will be data_root/ann_file               |
+| --data-prefix   | str   | The path to the image is {data_root}/{data_pefix}/{img_path}                                                                                                                         |
+| --output-dir    | str   | The path to save the visualization results. For devices that do not have a graphical interface, such as server clusters, the user can specify the output path to save the visualization results. |
+| --show-interval | float | The number of seconds between visualization images.                                                                                                                                  |
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:70]: incorrect params

File: docs/zh_cn/user_guides/visualization.md [hunk 5]
```diff
@@ -1 +1,108 @@
...
+下表为 `VisualizationHook` 支持的参数：
+
+| 参数            | 类型  | 说明                                                                                                 |
+| --------------- | ----- | ---------------------------------------------------------------------------------------------------- |
+| task            | str   | 算法任务, 可选择项为 'textdet', 'textrec', 'kie'                                                     |
+| data-root       | str   | 图像与标注存放的根目录                                                                               |
+| ann-file        | str   | 标注存放的目录。可以为相对路径和绝对路径， 如果为相对路径则最终的ann_file为 data_root/ann_file       |
+| --data-prefix   | str   | 图像的路径为{data_root}/{data_pefix}/{img_path}                                                      |
+| --output-dir    | str   | 可视化结果保存路径。对于不存在图形界面的设备，如服务器集群等，用户可以指定输出路径来保存可视化结果。 |
+| --show-interval | float | 可视化图像间隔秒数。                                                                                 |
```
Comment (xinke-wang) [inline docs/zh_cn/user_guides/visualization.md:70]: incorrect params

File: docs/en/user_guides/visualization.md [hunk 6]
```diff
@@ -1 +1,108 @@
...
+- MMOCR sets the visualization backend `vis_backend` to the local visualization backend `LocalVisBackend` by default, saving all visualization results and other training information in a local folder.
+
+## Storage
+
+MMOCR uses the local visualization backend [`LocalVisBackend`](mmengine.visualization.LocalVisBackend) by default, and the model loss, learning rate, model evaluation accuracy and visualization The information stored in `VisualizerHook` and `LoggerHook` will be saved to the `{work_dir}/{config_name}/{time}/{vis_data}` folder by default. In addition, MMOCR also supports other common visualization backends, such as `TensorboardVisBackend` and `WandbVisBackend`, and you only need to change the `vis_backends` type in the configuration file to the corresponding visualization backend. For example, you can store data to `TensorBoard` and `Wandb` by simply inserting the following code block into the configuration file.
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:36]:
```suggestion
MMOCR uses the local visualization backend [`LocalVisBackend`](mmengine.visualization.LocalVisBackend) by default, and the information stored in `VisualizerHook` and `LoggerHook`, including loss, learning rate, evaluation accuracy will be saved to the `{work_dir}/{config_name}/{time}/{vis_data}` folder. In addition, MMOCR also supports other common visualization backends, such as `TensorboardVisBackend` and `WandbVisBackend`, and you only need to change the `vis_backends` type in the configuration file to the corresponding visualization backend. For example, you can store data to `TensorBoard` and `Wandb` by simply inserting the following code block into the configuration file.
```

File: docs/en/user_guides/visualization.md [hunk 7]
```diff
@@ -1 +1,108 @@
...
+```Python
+_base_.Visualizer.vis_backends = [
+    dict(type='LocalVisBackend'),
+    dict(type='TensorboardVisBackend'),
+    dict(type='WandbVisBackend'),]
+```
+
+## Plot
+
+### Plot the prediction result information
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:47]:
```suggestion
### Plot the prediction results
```

File: docs/zh_cn/user_guides/visualization.md [hunk 8]
```diff
@@ -1 +1,108 @@
...
+
+```Python
+vis_backends = [dict(type='LocalVisBackend')]
+visualizer = dict(
+    type='TextxxxLocalVisualizer',  # 不同任务使用不同的可视化器
+    vis_backends=vis_backends,
+    name='visualizer')
+```
+
+依据以上示例，我们可以看出`Visualizer` 的配置主要由两个部分组成，即，`Visualizer`的类型以及其采用的可视化后端 `vis_backends`。
```
Comment (xinke-wang) [inline docs/zh_cn/user_guides/visualization.md:29]:
```suggestion
依据以上示例，我们可以看出 `Visualizer` 的配置主要由两个部分组成，即，`Visualizer`的类型以及其采用的可视化后端 `vis_backends`。
```

File: docs/zh_cn/user_guides/visualization.md [hunk 9]
```diff
@@ -1 +1,108 @@
...
+    dict(type='LocalVisBackend'),
+    dict(type='TensorboardVisBackend'),
+    dict(type='WandbVisBackend'),]
+```
+
+## 绘制
+
+### 绘制预测结果信息
+
+MMOCR 主要利用 [`VisualizationHook`](mmocr.engine.hooks.VisualizationHook)，绘制valuation 和 test 的预测结果, 默认情况下 `VisualizationHook`为关闭状态，默认配置如下：
```
Comment (xinke-wang) [inline docs/zh_cn/user_guides/visualization.md:49]:
```suggestion
MMOCR 主要利用 [`VisualizationHook`](mmocr.engine.hooks.VisualizationHook)，绘制 validation 和 test 的预测结果, 默认情况下 `VisualizationHook`为关闭状态，默认配置如下：
```

File: docs/en/user_guides/visualization.md [hunk 10]
```diff
@@ -1 +1,108 @@
...
+
+MMOCR mainly uses [`VisualizationHook`](mmocr.engine.hooks.VisualizationHook) to plot the prediction results of valuation and test, by default `VisualizationHook` is off, and the default configuration is as follows.
```
Comment (xinke-wang) [inline docs/en/user_guides/visualization.md:49]:
```suggestion
MMOCR mainly uses [`VisualizationHook`](mmocr.engine.hooks.VisualizationHook) to plot the prediction results of validation and test, by default `VisualizationHook` is off, and the default configuration is as follows.
```

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**